<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Raw Pixel Scraper</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background : #161219;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="urlInput">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>320</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #161219;
border: none;
border-bottom: 2px solid #BA1F33;
font: 15px;
color: 'white';
padding-bottom: 7px;</string>
    </property>
    <property name="placeholderText">
     <string>Raw Pixel URL</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pageInput">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>320</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #161219;
border: none;
font: 15px;
border-bottom: 2px solid #BA1F33;
color: 'white';
padding-bottom: 7px;</string>
    </property>
    <property name="placeholderText">
     <string>Number of Pages</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rawscraperDone">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>460</y>
      <width>250</width>
      <height>70</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border :4px solid '#BA1F33' ;
	border-radius: 15px ;
	font-size : 20px ;
	color: 'white' ;

	margin : 10px 20px ;
}

QPushButton:hover{
	background: '#BA1F33' ; 
	color: '#161219';
}</string>
    </property>
    <property name="text">
     <string>DONE</string>
    </property>
   </widget>
   <widget class="QLabel" name="smallLogo">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>0</y>
      <width>140</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../logo/small_logo_rb.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
